Readme_zh-CN.md

```diff
@@ -238,6 +238,24 @@ $ custom --list x,y,z
 [ 'x', 'y', 'z' ]
 ```
 
+### 必需的选项
+
+你可以使用`.requiredOption`指定一个必需的(强制性的)选项，这样的选项在命令行的命令中必须被给出，除非它拥有一个默认的值。另外，这个方法在格式上和使用`.option`一样：The method is otherwise the same as `.option` in format, 采用标志和说明，以及可选的默认值或自定义处理。
```

Comment:
I think this english text "The method is otherwise the same as `.option` in format," has been translated and can be deleted?

Reply:
Oh..You're right, I made a mistake